### PR TITLE
Completed css-two-column-layout challenge. (hopefully)

### PR DESCRIPTION
**Pull request by joshangeles**
image

### CODE REVIEW

File: css-two-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+}
+
+.col-half {
+  width: calc(100%/2);
+}
+
+.col-fourth {
+  width: 25%;
+  text-align: right;
+  padding-right: 0;
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:23]: `column` classes should only ever have `flex-basis`, nothing else. Anything additional necessary should be on a separate class.

File: css-two-column-layout/styles.css [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+}
+
+.col-three-fourths {
+  width: 75%;
+  align-items: right;
+  text-align: left;
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:29]: `column` classes should only ever have `flex-basis`, nothing else. Anything additional necessary should be on a separate class.

File: css-two-column-layout/styles.css [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+  margin-top: 3rem;
+  margin-bottom: 25%;
+  width: 250px;
+  height: 50px;
+  font-weight: bolder;
+  font-size: larger;
+}
+
+#learnmore {
+  margin-left: 425px;
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:84]: 😱 

There is a better way to do this, I'm sure. `text-align` typically is plenty to center a button element

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by joshangeles** [comment]
image

Removed unnecessary rules from column rulesets and changed the learn more button to not use a gross amount of margin.

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**